HyperwalletUISDK.podspec

```diff
@@ -11,6 +11,7 @@ Pod::Spec.new do |spec|
     spec.requires_arc          = true
     spec.swift_version         = '5.0'
     spec.dependency 'HyperwalletSDK', '1.0.0-beta05'
+    spec.dependency 'Insights', 'task/HW-57214-create-insights-framework'
```

Comment:
This will not work as it needs a tag here  - You can verify this by running 
`pod lib lint --sources='https://github.com/hyperwallet/hyperwallet-ios-sdk-specs.git','https://github.com/CocoaPods/Specs.git', --allow-warnings --skip-tests --verbose --no-clean`

Reply:
We have already updated `hyperwallet-ios-sdk-specs` to reflect these changes

Reply:
Refer to https://guides.cocoapods.org/making/private-cocoapods.html

Reply:
Ok, Thank You